include/wlc/wlc.h

```diff
@@ -364,6 +371,15 @@ bool wlc_output_set_views(wlc_handle output, const wlc_handle *views, size_t mem
 /** Focus output. Pass zero for no focus. */
 void wlc_output_focus(wlc_handle output);
 
+/** Get output modes */
+const struct wlc_output_mode* wlc_output_get_modes(wlc_handle output, size_t *out_memb);
```

Comment:
Presumably this structure has a limited lifetime of validity. Can you speak to that in the docs?